### PR TITLE
renaming codebase to gitsourceref

### PR DESCRIPTION
**Pull request by tinakurian**

### CODE REVIEW

**Review by sbose78 — APPROVED** [review]
Good catch, thanks!

File: pkg/apis/devconsole/v1alpha1/component_types.go [hunk 1]
```diff
@@ -4,24 +4,24 @@ import (
...
-// ComponentSpec defines the desired state of Component
+// ComponentSpec defines the desired state of Component.
 // +k8s:openapi-gen=true
 type ComponentSpec struct {
 	// Container image use to build (nodejs, golang etc..)
 	BuildType string `json:"buildType"`
-	// Codebase is the source code of your component. Atm only public remote URL are supported.
-	Codebase string `json:"codebase"`
+	// GitSourceRef is the source code of your component. Currently, only public remote URLs are supported.
+	GitSourceRef string `json:"codebase"`
```
Comment (alexeykazakov) [inline pkg/apis/devconsole/v1alpha1/component_types.go:13]: You need to change the json tag too.

Comment (alexeykazakov) [reply pkg/apis/devconsole/v1alpha1/component_types.go:13]: https://github.com/redhat-developer/devconsole-api/pull/12

Comment (sbose78) [reply pkg/apis/devconsole/v1alpha1/component_types.go:13]: Thanks a lot, I missed. :(

**Pull request merged**